crates/eww/src/config/system_stats.rs

```diff
@@ -7,32 +7,49 @@ use sysinfo::{ComponentExt, DiskExt, NetworkExt, NetworksExt, ProcessorExt, Syst
 
 static SYSTEM: Lazy<Mutex<System>> = Lazy::new(|| Mutex::new(System::new()));
 
-pub fn disk() -> String {
+pub fn get_disks() -> String {
     let mut c = SYSTEM.lock().unwrap();
     c.refresh_disks_list();
 
     format!(
         "{{ {} }}",
         c.get_disks()
             .iter()
-            .map(|c| format!(
-                r#""{}": {{"name": {:?}, "total": {}, "free": {}}}"#,
-                c.get_mount_point().display(),
-                c.get_name(),
-                c.get_total_space(),
-                c.get_available_space(),
-            ))
+            .map(|c| {
+                let total_space = c.get_total_space();
+                let available_space = c.get_available_space();
+                format!(
```

Comment:
Given that we're providing percentages for ram now, would make sense to also have a `used_perc` here